addon/session-stores/cookie.js

```diff
@@ -143,8 +166,7 @@ export default BaseStore.extend({
   },
 
   _read(name) {
-    let value = document.cookie.match(new RegExp(`${name}=([^;]+)`)) || [];
-    return decodeURIComponent(value[1] || '');
+    return this.get('_cookies').read(name) || '';
```

Comment:
I ran into this... the solution (which might have been rebased out) was to manually inject the cookies service. Did the rebasing mess that up?

https://github.com/simplabs/ember-simple-auth/blob/2d49133ec8543ae53af713d7551072ba49f65ce7/addon/initializers/setup-session.js#L17